src/safeds/ml/nn/layers/_internal_layers.py

```diff
@@ -56,6 +56,18 @@ def forward(self, x: Tensor) -> Tensor:
         return self._fn(self._layer(x))
 
 
+class _InternalDropoutLayer(nn.Module):
+    def __init__(self, propability: float) -> None:
```

Comment:
```suggestion
    def __init__(self, probability: float) -> None:
```

Use a rename refactoring.